packages/block-editor/src/components/block-list/block-list-item.native.js

```diff
@@ -98,7 +98,7 @@ function BlockListItemContent( {
 			const name = getBlockName( clientId );
 			const parentName = getBlockName( rootClientId );
 			const { align } = getBlockAttributes( clientId ) || {};
-			const { align: parentBlockAlign } =
+			const { textAlign: parentBlockAlign } =
```

Comment:
@carolinan, thinking on this more, I realised we may need to co-ordinate releases to ensure centred quote blocks don't break for app users. Would you be able to hold off from merging until I co-ordinate a release plan with the mobile team? I'll update here when we have something in place. Thank you!

Reply:
@carolinan, thank you for your patience here! The earliest app release we'd be able to get this into is `24.4`, which is set to be released to users on Monday, 18th March. With that in mind, could you merge this _after_ the `17.8` Gutenberg release is cut this coming Wednesday? If this is included in the `17.9` release (set to be released on March 13th), it'd minimise the number of days it's not available for app users. 

From my testing, I confirmed that having the change in place on the web but not in the app won't break the site’s content. The only issue I found is that the app's in-editor preview won’t reflect whether a Quote block is centred or right-aligned, which might be confusing. However, this will only be an issue for blocks that are first saved on the web then opened in the app. The in-app alignment buttons will continue to work and correct alignment will still be visible on the site’s front-end.

With the above in mind, we think minimising the number of days that the change is available on one platform but not another is the best option. Let me know if you're okay with delaying merge until after this Wednesday, for the `17.9` release. Thanks!

Reply:
Yes, it can wait.

Reply:
@carolinan, just to follow up, if you're able to merge this before this coming Thursday, we'll  then work to get it into the app's `24.4` release. Thanks again for all your help here!